.github/infrastructure/docker-compose-postgresql.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
+services:
+  db:
+    image: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
```

Comment:
It's not really to big a deal since this is just for test, but I don't know if there's a security guideline for not including passwords in committed config files. @artursouza, do you have an opinion?

If you want an example for dynamically populating this values with a temporary secret, https://github.com/dapr/components-contrib/pull/1115 also has an example on how to do that.

Reply:
I think it's fine since it's for test only. The hardcoded plain-text password also appears in other docker compose file.